app/models/info_request.rb

```diff
@@ -254,7 +254,7 @@ def self.guess_by_incoming_email(*emails)
   def self._guess_idhash_from_email(incoming_email)
     incoming_email = incoming_email.downcase
     incoming_email =~ /request\-?(\w+)-?(\w{8})@/
-    id = $1
+    id = $1.to_i
```

Comment:
Will this suffer from the same problem as https://github.com/mysociety/alaveteli/issues/4996?

Reply:
It will, it's been copied from the `self. _extract_id_hash_from_email` so it will fail identically - I'll rebase and switch to the #4997 version once I've got that merged in

Reply:
(or possibly lightly refactor so we attempt to transform Ls and Os and see if Integer conversion is effective in one reusable place, TBC)